doc/oca_epak_wsdl.xml

```diff
@@ -0,0 +1,2208 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
@cavi21 I think we already have a copy of this file here: https://github.com/ombulabs/oca-epak/blob/master/doc/oca_wsdl_docs.xml, maybe yours is a bit more updated and also has a more accurate name, could you remove the one I linked? Thanks!